src/anonymize_proxy.js

```diff
@@ -1,7 +1,7 @@
-import { Server } from './server';
-import {
+const { Server } = require('./server');
```

Comment:
does it really make sense to do esm -> cjs conversion just to get rid of babel, when we plan to rewrite to ts with esm in the following step?

Reply:
babel didn't like node.js streams for some reason. it called some function that did not exist (don't remember exactly what function was being called). maybe it needed more configuration, for me it was just easier to drop babel. only es imports were used.

`proxy-chain` doesn't use esm yet https://cdn.jsdelivr.net/npm/proxy-chain@1.0.4/build/index.js

Reply:
i am talking about the source code, it uses esm style imports. and once we make this TS, we will use them again. not talking about what gets compiled/published.

but sure, if there were real issue with babel, lets get rid of it

Reply:
> i am talking about the source code, it uses esm style imports. and once we make this TS, we will use them again.

that's right.

> if there were real issue with babel, lets get rid of it

it generated something like this

```js
    static forward(request, response) {
        return _asyncToGenerator(function* () {
            var pipeline = util.promisify(stream.pipeline);

            yield pipeline(got.stream(request.url, {
                method: request.method,
                headers: request.headers,
                decompress: false,
                followRedirect: false
            }), response);
        })();
    }
```

and `_asyncToGenerator` was undefined.